app/lib/api_support/forbid_params.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module ApiSupport
+  class UnpermittedParameters < StandardError
```

Comment:
Just a nitpick, unless you have special considerations, to be `UnpermittedParametersError`.

Reply:
Done